agents/plugins/smart

```diff
@@ -62,6 +62,43 @@ megaraid_info()
        done
 }
 
+storcli_info()
+{
+    $1 /call show all | while read -r line; do
+        case "$line" in
+            "Controller = "*)
+                CTRL=$( echo "$line" | cut -d ' ' -f 3 )
+            ;;
+            "PCI Address = "*)
+                PCIA=$( echo "$line" | cut -d ' ' -f 4 | (IFS=: read -r _d _c _b _a; echo ${_a}${_b}:$_c:$_d) )
```

Comment:
I'm not sure about this method to determine the PCI bus address.
It works for us so far, but we aren't sure, it's guaranteed to work yet.